test/xUnit/csharp/test_PSConfiguration.cs

```diff
@@ -74,8 +74,8 @@ public PowerShellPolicyFixture()
                 ScriptBlockLogging = new ScriptBlockLogging() { EnableScriptBlockInvocationLogging = true, EnableScriptBlockLogging = false },
                 ModuleLogging = new ModuleLogging() { EnableModuleLogging = false, ModuleNames = new string[] { "PSReadline", "PowerShellGet" } },
                 ProtectedEventLogging = new ProtectedEventLogging() { EnableProtectedEventLogging = false, EncryptionCertificate = new string[] { "Joe" } },
-                Transcription = new Transcription() { EnableInvocationHeader = true, EnableTranscripting = true, OutputDirectory = "c:\tmp" },
-                UpdatableHelp = new UpdatableHelp() { DefaultSourcePath = "f:\temp" },
+                Transcription = new Transcription() { EnableInvocationHeader = true, EnableTranscripting = true, OutputDirectory = "c:\\tmp" },
+                UpdatableHelp = new UpdatableHelp() { DefaultSourcePath = "f:\\temp" },
```

Comment:
Perhaps verbatim string literal would be more readable. `@"f:\temp"`

@powercode Many thanks for fixing this!